esp/mlrs-wifi-bt-bridge/mlrs-wifi-bt-bridge.ino

```diff
@@ -75,23 +68,23 @@ List of supported modules, and board which need to be selected
 
   int port_tcp = 5760; // connect to this port per TCP // MissionPlanner default is 5760
   int port_udp = 14550; // connect to this port per UDP // MissionPlanner default is 14550
-#endif
 
+  // WiFi channel
+  // 1 is the default, 13 (2461-2483 MHz) has the least overlap with mLRS 2.4 GHz frequencies.
+  // Note: Channel 13 is generally not available in the US, where 11 is the maximum.
+  int wifi_channel = 13;
+
+  // WiFi power
+  // comment out for default setting
+  // Note: In order to find the possible options, right click on WIFI_POWER_19_5dBm and choose "Go To Definiton"
+  #define WIFI_POWER  WIFI_POWER_2dBm // WIFI_POWER_MINUS_1dBm is the lowest possible, WIFI_POWER_19_5dBm is the max
+#elif (PROTOCOL_TYPE == 2) // Bluetooth
```

Comment:
don't like PROTOCOL_TYPE :)
since you renamed files to wireless, WIRELESS_PROTOCOL suggests itself

Reply:
Done in latest commit.